network/model.go

```diff
@@ -11,3 +11,8 @@ type CreateInteractionRequest struct {
 type CreateGameRequest struct {
 	PlayerIDs []model.PlayerID `json:"playerIDs"`
 }
+
+type CreatePlayerRequest struct {
```

Comment:
So if we're not using the backend models to receive or return from/to the web client, I guess we need to define types that are "safe" to be passed over the wire?